admin/tools/docker/deployment/in2p3/env.example.sh

```diff
@@ -30,8 +42,29 @@ HOST_DATA_DIR=/qserv/data
 # Log directory location on docker host, optional
 HOST_LOG_DIR=/qserv/log
 
-CONTAINER_NAME=qserv                                 # Do not edit
+# Advanced configuration
+# ======================
 
-# shmux access
-export PATH="$PATH:/opt/shmux/bin"
+# Return a hostname list
+# arg1: format of hostnames
+# arg2: id of first host
+# arg3: id of last host, optional
+printHostname () {
+    if [ -n "$3" ]; then
+		local_last="$3"
+	else
+	    local_last="$2"
+    fi
```

Comment:
Indentation messed up in these lines. Do not use TABs, or do not mix TABs with spaces.